packages/taraxa-ui/src/components/BarChart/BarChart.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import BarChart from './BarChart';
+
+export default {
+  title: 'Components/BarChart',
+  component: BarChart,
+  argTypes: {},
+} as Meta;
+
+const Template: Story<any> = (args) => <BarChart {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  labels: ['33366', '33367', '33368', '33369'],
```

Comment:
On vertical barcharts these two values should be the opposite: values on the X axis and labels on the Y...